frontend_csv/src/parser/ast_msg.rs

```diff
@@ -142,6 +145,12 @@ pub fn map_csvtoken(v: Vec<CSVTokenType>) -> AstMsgLineType {
         [CSVTokenType::MsgData, CSVTokenType::LiteralString, CSVTokenType::LiteralString, CSVTokenType::Byte, CSVTokenType::LiteralString] => {
```

Comment:
Can you please simplify the following code? maybe by splitting it in different steps?

```
   _ if [CSVTokenType::U16, CSVTokenType::U32, CSVTokenType::U64]
            .iter()
            .any(|item| v == [test_msgdatalen_vec.as_slice(), [*item].as_slice()].concat()) =>
        {
            AstMsgLineType::MsgdataLength
        }
```

Reply:
I need to check if it can.... because there is two loop here...